packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -698,16 +754,15 @@ export class ListBase extends CollectionWidget<ListBaseProperties> {
       return false;
     }
 
-    const $content = this._scrollView.content();
-    const $container = this._scrollView.container();
+    const $content = this._scrollView?.content();
```

Comment:
ScrollView is rendered on initialization, so we can use '!' mark in _scrollview definition to be sure that it exist